### DISCUSSION TITLE
SoftConsole versions 2021.3 and 2022.2 removed from Microchip website

### DISCUSSION THREAD

**Discussion by artruder** [polarfire-soc/polarfire-soc-documentation#220] [General] [answered]
Hi,

were the latest versions of SoftConsole removed from the Microchip website (https://www.microchip.com/en-us/products/fpgas-and-plds/fpga-and-soc-design-tools/soc-fpga/softconsole)?

Under latest release only version 2021.1 is listed.

![image](https://user-images.githubusercontent.com/34745788/196381970-25113d33-9d85-49da-a3ba-b49a5f26c6e5.png)

However, I have version 2021.3 and 2022.2 installed and wanted to install 2022.2 on a new PC but cannot find the installer anymore.

Is this intentional and I should switch to 2021.1?

Thanks in advance for any support.

**Comment by hughbreslin** [comment, marked as answer]
Hi @artruder we unfortunately have to do a re-release of SoftConsole due to a license issue (there is no functional issue) which meant we pulled the installers, this is currently working its way through to a release.

You should be able to do everything you need with 2021.1, if there is something specific to 2022.2 that you need it should be available shortly :)

**Reply by artruder** [reply]
Thanks for the clarification!

**Reply by hughbreslin** [reply]
@artruder SoftConsole 2022.2 is now available from the Microchip [website](https://www.microchip.com/en-us/products/fpgas-and-plds/fpga-and-soc-design-tools/soc-fpga/softconsole) :)
